src/test/java/tds/exam/results/validation/impl/XSDBackedTDSReportValidatorTest.java

```diff
@@ -0,0 +1,100 @@
+package tds.exam.results.validation.impl;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.xml.sax.SAXException;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import java.io.IOException;
+
+import tds.exam.results.configuration.ExamResultsTransmitterServiceProperties;
+import tds.exam.results.trt.TDSReport;
+import tds.exam.results.validation.TDSReportValidator;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+public class XSDBackedTDSReportValidatorTest {
+    private Resource resource;
+    private ExamResultsTransmitterServiceProperties properties;
+
+    @Before
+    public void setUp() throws Exception {
+        resource = new ClassPathResource("xsd/TestResultsTransmissionFormat_Schema.xsd");
+        properties = new ExamResultsTransmitterServiceProperties();
+        properties.setValidateTrtXml(true);
+    }
+
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    @Test
+    public void shouldNotValidate() throws IOException, SAXException {
+        JAXBContext context = mock(JAXBContext.class);
+        properties.setValidateTrtXml(false);
+        TDSReportValidator validator = new XSDBackedTDSReportValidator(context, resource, properties);
+
+        TDSReport report = new TDSReport();
+
+        validator.validateReport(report);
+
+        verifyZeroInteractions(context);
+    }
+
+    @Test
+    @Ignore
```

Comment:
They do provide a specific set of possible values for many of the attributes, but not sure if all of them are covered. Unfortunately I don't believe XJC generates the actual Java enums or constants out of the box. I agree this can be revisited later in the near future.